recipes/libusb/all/conanfile.py

```diff
@@ -57,13 +57,6 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
-    def validate(self):
-        if self.options.get_safe("enable_udev"):
-            if self.settings.os == "Android":
-                raise ConanInvalidConfiguration("udev can't be enabled for Android yet, since libudev recipe is missing in CCI.")
-            if tools.cross_building(self):
-                raise ConanInvalidConfiguration("udev can't be enabled yet if cross-compiling")
-
     def build_requirements(self):
```

Comment:
```suggestion
    def validate(self):
        if self.options.get_safe("enable_udev"):
            if self.settings.os == "Android":
                raise ConanInvalidConfiguration("udev can't be enabled for Android yet, since libudev recipe is missing in CCI.")

    def build_requirements(self):
```
on Android, "libudev/system" is not part of the requirements of libusb